src/components/Grid/Column.js

```diff
@@ -13,8 +13,9 @@ const validBreakpoints = [
   'max-large',
 ];
 
-const breakPointProp = breakpoint => `${breakpoint}-sizeOf`;
+const breakPointProp = (breakpoint, suffix) => `${breakpoint}-${suffix}`;
 const sizeRegex = /^([1-9]|1[0-2])-([1-9]|1[0-2])$/;
+const orderRegex = /^((1[0,1,2])|[1-9])$/;
```

Comment:
the regex is rather simple in what it does, maybe a functional condition with Math.min/Math.max would be better here? You could keep the regex for consistency's sake, but the function will be more performant.

Reply:
The prop can be a string as well as a number. I could use parseInt and isNaN to check the prop. Are regex really that slow?

Reply:
We can keep it here, but in general, JS regexes are slower by several orders of magnitude than other operations